### ISSUE TITLE
multiple packages sr fails

### ISSUE THREAD

**Issue by mcepl** [openSUSE/osc#541]
I have a directory with a checked out project consisting of two packages. Then, this follows:
```
home:mcepl:pydaemon$ isc sr --cleanup -s 191442 -s 191443 SUSE:SLE-12-SP5:GA
Submitting package  python-lockfile.SUSE_SLE-12-SP5_GA
Submitting package  python-python-daemon.SUSE_SLE-12-SP5_GA
Request created:  191640 Traceback (most recent call last):
  File "/home/matej/.local/bin/osc", line 11, in <module>
    load_entry_point('osc===0.165.git', 'console_scripts', 'osc')()
  File "/home/matej/.local/lib/python3.7/site-packages/osc-0.165.git-py3.7.egg/osc/babysitter.py", line 204, in main
  File "/home/matej/.local/lib/python3.7/site-packages/osc-0.165.git-py3.7.egg/osc/babysitter.py", line 62, in run
  File "/home/matej/.local/lib/python3.7/site-packages/osc-0.165.git-py3.7.egg/osc/cmdln.py", line 344, in main
    line = 'EOF'
  File "/home/matej/.local/lib/python3.7/site-packages/osc-0.165.git-py3.7.egg/osc/cmdln.py", line 367, in cmd
    if self.use_rawinput and self.completekey:
  File "/home/matej/.local/lib/python3.7/site-packages/osc-0.165.git-py3.7.egg/osc/cmdln.py", line 501, in onecmd
    else:
  File "/home/matej/.local/lib/python3.7/site-packages/osc-0.165.git-py3.7.egg/osc/cmdln.py", line 1232, in _dispatch_cmd
    inst.optparser = inst.get_optparser()
  File "/home/matej/.local/lib/python3.7/site-packages/osc-0.165.git-py3.7.egg/osc/commandline.py", line 1311, in do_submitrequest
UnboundLocalError: local variable 'result' referenced before assignment
home:mcepl:pydaemon$
```

**Comment by mcepl** [comment]
This was probably just an incorrect command. Error messages suck, but that's probably not that important.

**Comment by lethliel** [comment]
The error message needs improvement. So I will reopen this.

**Comment by dmach** [comment]
This seems to be fixed in 3da7b9b3d8d2f46ae0ce1d9f84bd8c1666335603 already. Closing.
